core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3108,6 +3132,14 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String SWIFT_USER_KEY = "fs.swift.user";
     public static final String SWIFT_USE_PUBLIC_URI_KEY = "fs.swift.use.public.url";
     public static final String SWIFT_REGION_KEY = "fs.swift.region";
+    public static final String UNDERFS_COS_CONNECT_MAX = "alluxio.underfs.cos.connection.max";
+    public static final String UNDERFS_COS_CONNECT_TIMEOUT =
+        "alluxio.underfs.cos.connection.timeout";
+    public static final String UNDERFS_COS_SOCKET_TIMEOUT = "alluxio.underfs.cos.socket.timeout";
+    public static final String COS_ACCESS_KEY = "fs.cos.accessKeyId";
```

Comment:
@fastio is this required to be `fs.cos.accessKeyId`? it will work better if is is named after `fs.cos.access.key` for dockerized environment. If you think it better to be `fs.cos.accessKeyId` for some conventional reasons, i can still make it work. Just needs more tuning in our docker side work.

Similar questions for `fs.cos.accessKeySecret`, can we call it `fs.cos.secret.key`? 
and also for `fs.cos.appId` to be `fs.cos.app.id`?

Reply:
The property name is not required to be special style like fs.cos.abcXyz.  It's no problem to change it to fs.cos.access.key.  The same is true for fs.cos.accessKeySecret and fs.cos.appId.